$NetBSD: patch-.cargo_config.toml,v 1.1 2024/07/30 14:09:35 vins Exp $

Do not force static linking;
prevents conflicting types with system's pcre2.

--- .cargo/config.toml.orig	2024-07-30 11:44:48.648649075 +0000
+++ .cargo/config.toml
@@ -1,2 +1,2 @@
 [env]
-PCRE2_SYS_STATIC = "1"
+PCRE2_SYS_STATIC = "0"
